parser.go

```diff
@@ -100,6 +100,12 @@ func moveToRoom(exit string) {
 		fmt.Printf("Make sure you CLIMB DOWN before you try to go anywhere!\n")
 		return
 	}
+
+	if eagleWatching {
```

Comment:
Should we have a function that constantly checks if the player is in the yard and if they are if the eagle is watching. If the player is in the yard and the eagle is watching this should print, not just when the player tries to leave the room. I'm not sure where this function would go though because it's not part of moveToRoom, in that case.

Reply:
Moving this to issues.